vendor/vendor.json

```diff
@@ -2,6 +2,156 @@
 	"comment": "",
 	"ignore": "test",
 	"package": [
+		{
```

Comment:
not something we need to take up in this PR, but it seems odd that go dependencies are committed, but JS dependencies are installed.